### PR TITLE
Batch fix sonarqube failures

### PR DESCRIPTION
**Pull request by mximp**
#1897 Fixes of SonarCloud findings.
Add puzzles for the rest.

### CODE REVIEW

**Comment by mximp** [comment]
@yegor256 please reveiw

File: eo-runtime/src/main/java/org/eolang/PhDefault.java [hunk 1]
```diff
@@ -284,6 +284,16 @@ public String locator() {
         return "?";
     }
 
+    /**
+     * Clean up resources.
+     * This includes call of {@link ThreadLocal#remove()} method to prevent
+     * memory leaks.
+     */
+    public static void cleanUp() {
```
Comment (yegor256) [inline eo-runtime/src/main/java/org/eolang/PhDefault.java:292]: @mximp I don't see anyone calling this method. I think, `remove()` is supposed to be called right after we use `TERMS`, not in a separate method. But I'm not sure...

Comment (mximp) [reply eo-runtime/src/main/java/org/eolang/PhDefault.java:292]: @yegor256 You are right - the method is not called by anyone. The thing is I cannot define the moment "right after TERM is used". It's static and holds the value which is required throughout the program execution. So I see correct place for `remove` to be called before program exits. But that doesn't make sense.
So in this case I just created `fake` method according to SonarQube expectations.

**Comment by mximp** [comment]
@yegor256 please check my comment https://github.com/objectionary/eo/pull/1930#discussion_r1144887719

**Comment by mximp** [comment]
@yegor256 reminder

**Comment by yegor256** [comment]
@rultor merfe

**Comment by mximp** [comment]
@yegor256 please try again

**Comment by mximp** [comment]
@yegor256 please merge

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**